src/Dfe.PlanTech.Infrastructure.Data/PlanTechDbContext.cs

```diff
@@ -132,4 +132,32 @@ public Task<int> ExecuteRaw(FormattableString sql, CancellationToken cancellatio
     public Task<T?> FirstOrDefaultAsync<T>(IQueryable<T> queryable, CancellationToken cancellationToken = default) => queryable.FirstOrDefaultAsync(cancellationToken);
 
     public Task<List<T>> ToListAsync<T>(IQueryable<T> queryable, CancellationToken cancellationToken = default) => queryable.ToListAsync(cancellationToken);
+
+    public async Task<List<Answer>> GetAnswersForLatestSubmissionBySectionId(string sectionId, int establishmentId)
+    {
+        var latestSubmissionIdQuery = Submissions
+            .Where(s => s.SectionId == sectionId && s.EstablishmentId == establishmentId && s.Completed)
+            .OrderByDescending(s => s.DateCreated)
+            .Select(s => s.Id)
+            .FirstOrDefaultAsync();
+
+        if (latestSubmissionIdQuery == null)
+        {
+            return null;
+        }
+
+        var latestSubmissionId = await latestSubmissionIdQuery;
+
+        var answersQuery = Responses
```

Comment:
Have you tried merging this query with the `latestSubmissionIdQuery`? You should be able to execute it in one query, but I'm unsure as to whether it would be more or less performant.